extensions/iceberg/src/main/java/io/deephaven/iceberg/util/IcebergTools.java

```diff
@@ -30,18 +30,20 @@ public static IcebergCatalogAdapter createAdapter(
      * <p>
      * The minimal set of properties required to create an Iceberg catalog are:
      * <ul>
-     * <li>{@code "catalog-impl"} or {@code "type"} - the Java catalog implementation to use. When providing
-     * {@code "catalog-impl"}, the implementing Java class should be provided (e.g.
-     * {@code "org.apache.iceberg.rest.RESTCatalog"} or {@code "org.apache.iceberg.aws.glue.GlueCatalog")}. Choices for
-     * {@code "type"} include {@code "hive"}, {@code "hadoop"}, {@code "rest"}, {@code "glue"}, {@code "nessie"},
-     * {@code "jdbc"}.</li>
-     * <li>{@code "uri"} - the URI of the catalog.</li>
+     * <li>{@value CatalogProperties#CATALOG_IMPL} or {@value CatalogUtil#ICEBERG_CATALOG_TYPE} - the Java catalog
+     * implementation to use. When providing {@value CatalogProperties#CATALOG_IMPL}, the implementing Java class should
+     * be provided (e.g. {@code "org.apache.iceberg.rest.RESTCatalog"} or
+     * {@code "org.apache.iceberg.aws.glue.GlueCatalog")}. Choices for {@value CatalogUtil#ICEBERG_CATALOG_TYPE} include
+     * {@value CatalogUtil#ICEBERG_CATALOG_TYPE_HIVE}, {@value CatalogUtil#ICEBERG_CATALOG_TYPE_HADOOP},
+     * {@value CatalogUtil#ICEBERG_CATALOG_TYPE_REST}, {@value CatalogUtil#ICEBERG_CATALOG_TYPE_GLUE},
+     * {@value CatalogUtil#ICEBERG_CATALOG_TYPE_NESSIE}, {@value CatalogUtil#ICEBERG_CATALOG_TYPE_JDBC}.</li>
+     * <li>{@value CatalogProperties#URI} - the URI of the catalog.</li>
      * </ul>
      * <p>
      * Other common properties include:
      * </p>
      * <ul>
-     * <li>{@code "warehouse"} - the location of the data warehouse.</li>
+     * <li>{@value CatalogProperties#WAREHOUSE_LOCATION} - the location of the data warehouse.</li>
      * <li>{@code "client.region"} - the region of the AWS client.</li>
      * <li>{@code "s3.access-key-id"} - the S3 access key for reading files.</li>
      * <li>{@code "s3.secret-access-key"} - the S3 secret access key for reading files.</li>
```

Comment:
In the spirit of this refactoring
```suggestion
     * <li>{@value AwsClientProperties.CLIENT_REGION} - the region of the AWS client.</li>
     * <li>{@value S3FileIOProperties.ACCESS_KEY_ID} - the S3 access key for reading files.</li>
     * <li>{@value S3FileIOProperties.SECRET_ACCESS_KEY} - the S3 secret access key for reading files.</li>.           
      * <li>{@code S3FileIOProperties.ENDPOINT} - the S3 endpoint to connect to.</li>
```

Reply:
These references are not available on the classpath form this context; technically, the Iceberg layer does _not_ depend on S3.